### PR TITLE
Replace slack reference for questions with discord server

### PR DESCRIPTION
**Pull request by Neppord**
I tried as much as I could to transfer the sentiment from the old text to the new

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Everyone is welcome! People of all experience levels can join, begin contributin
 Feel free to ask questions by opening an issue on the relevant library. Maintainers are also active on:
 
 - The [PureScript Discourse](https://discourse.purescript.org) (the most popular option and best for detailed questions)
-- The [Functional Programming Slack](https://functionalprogramming.slack.com) ([link to join](https://fpchat-invite.herokuapp.com)!) in the `#purescript` and `#purescript-beginners` channels (best for quick, informal questions)
+- The [Purescript Discord](https://purescript.org/chat) in the `#help-language`, `#help-code` and `#help-tools` channels (best for quick, informal questions)
```
Comment (thomashoneyman) [inline contributing.md]: I’d prefer we avoid naming specific channels in case they change over time (which they have already!) but otherwise 👍

Comment (Neppord) [reply contributing.md]: good point, I still think its better then it was, maybe I could have fixed it with a new PR, to keep up the flow :).

**Review by thomashoneyman — APPROVED** [review]

**Comment by Neppord** [comment]
The check fails due to wrong purescript version. I have not touched the version, how do I fix it?

**Comment by JordanMartinez** [comment]
It's not the version of PureScript, it's the package set being used. Run `spago upgrade-set` in the correct folder and that should fix the issue.

**Comment by Neppord** [comment]
@JordanMartinez thanks I'll do that next time I have the same issue!

**Pull request merged**